pyproject.toml

```diff
@@ -105,6 +105,16 @@ docs = [
     "tomli; python_version < '3.11'",
     "sphinxcontrib-globalsubs >= 0.1.1",
 ]
+dev = [
+    "astropy[recommended]",  # installs the most common optional dependencies
+    "astropy[test]",  # installs the [test] dependencies
+    "astropy[docs]",  # installs the [docs] dependencies
+    "astropy[typing]",  # installs the [typing] dependencies
+]
+dev_all = [
```

Comment:
These new installation options should be (briefly) introduced somewhere in the developer documentation.

Reply:
Done.

Reply:
Done.